lib/membrane_rtc_engine/endpoints/webrtc/variant_selector.ex

```diff
@@ -142,6 +142,15 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.VariantSelector do
     }
   end
 
+  @doc """
+  Updates bitrate assigned to a variant in the selector. Expected bitrate unit is kbps.
+  """
+  @spec update_variant_bitrate(t(), Track.variant(), non_neg_integer()) :: t()
+  def update_variant_bitrate(%__MODULE__{} = selector, variant, bitrate) do
+    variant_bitrates = %{selector.variant_bitrates | variant => bitrate * 1024}
+    %{selector | variant_bitrates: variant_bitrates}
```

Comment:
This is not enough. You need to update an allocation, if you received an update about the currently selected or queued variant, or update requested bitrate if this affects your `next_desired_variant`